content/user/ui/settings/Integrations.md

```diff
@@ -110,44 +113,42 @@ If integration with an unsupported system is needed, this can be accomplished in
 
 Method | Description
 ------ | -----------
-**Integrate With The TeamServer REST API** | You can find complete documentation here: [REST API](dev_api1.html#intro)
+**Integrate With The Contrast REST API** | You can find complete documentation here: [REST API](dev_api1.html#intro)
 **Export Findings To CSV** | Findings can be exported to **Comma-Separated Values (CSV)** and imported into a bugtracker of your choice.
 **Export Findings To XML** | Individual or groups of findings can be exported to **XML** format and imported into a bugtracker of your choice.
 
 
 ---
 
-## Configuring Atlassian JIRA
+## JIRA Integration
+Integrating JIRA with Contrast allows you to automatically generate tickets, synchronize comments and push notifications for your applications.
 
 ### Prerequisites
 
 * JIRA account credentials (username and password)
-* The user must have permission to create issues in the target project
-* A running JIRA instance accessible via HTTP to the Contrast TeamServer
+* User permission to create issues in the target project
```

Comment:
To me this wording reads as if creating an issue needs the user's permission, rather than that the user needs the correct permissions. :P Could just say "Permission to create..." or "Appropriate permissions in place to be able to create..." ?